roles/k8s-state/templates/vault.list.j2

```diff
@@ -0,0 +1,4 @@
+# {{ ansible_managed }}
```

Comment:
I don't know the file extension "j2". What is it for ?

Reply:
Ansible uses [Jinja2](https://pypi.org/project/Jinja2/) as templating engine. The extension `.j2` does not really exists and is only useful for clarity purposes.